tests/utils/test_helpers.py

```diff
@@ -213,7 +214,10 @@ def test_is_in(self):
         # python caches small integers, so i is 3 will be True,
         # but `big_i is 2 ** 31` is False.
         self.assertTrue(helpers.is_in(big_i, test_container))
-        self.assertFalse(helpers.is_in(2 ** 31, test_container))
+        if sys.version_info.minor < 7:
+            self.assertFalse(helpers.is_in(2 ** 31, test_container))
+        else:
+            self.assertTrue(helpers.is_in(2 ** 31, test_container))
```

Comment:
This seems like a change in behaviour. What's going on here?

Reply:
Yes, the big int caching behavior is changed in py 3.7. multiple creation of `2**31`s in this test is now returning the same reference id.

Reply:
Give me a second, we don't use this helper anywhere other than tests, I'll remove it on master then this issue just goes away.

Reply:
https://issues.apache.org/jira/browse/AIRFLOW-5276 (separate PR - you're welcome to take that one on, I wont get around to it today.)

Reply:
I will take a stab at that today. Deleting unused code is one of my fav exercise :)

Reply:
PR sent: https://github.com/apache/airflow/pull/5878